examples/pylab_examples/custom_cmap.py

```diff
@@ -55,7 +67,30 @@
 never used.
 
 """
+# Make some illustrative fake data:
+
+x = np.arange(0, np.pi, 0.1)
+y = np.arange(0, 2*np.pi, 0.1)
+X, Y = np.meshgrid(x, y)
+Z = np.cos(X) * np.sin(Y) * 10
+
+
+# --- Colormaps from a list ---
 
+colors = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # R -> G -> B
+n_bins = [2, 5, 10, 100]  # Discretizes the interpolation into bins
+cmap_name = 'my_list'
+f, axs = plt.subplots(1, 4, figsize=(10, 5))
```

Comment:
The `cax` kwarg takes an Axes that you have made yourself to hold the colorbar; I think what you want is the `ax` kwarg, which takes the parent Axes from which space for the colorbar will be taken.